internal/middleware/perfinsights/perfinsights_test.go

```diff
@@ -96,7 +96,7 @@ func TestObserveShapeLatency(t *testing.T) {
 		require.Equal(t, io_prometheus_client.MetricType_HISTOGRAM, metric.GetType())
 
 		require.Equal(t, uint64(1), metric.GetMetric()[0].Histogram.GetSampleCount())
-		require.Equal(t, float64(0.1), metric.GetMetric()[0].Histogram.GetSampleSum())
+		require.InEpsilon(t, 0.1, metric.GetMetric()[0].Histogram.GetSampleSum(), 0.01)
```

Comment:
please reframe from using InEpsilon here

Reply:
Noted and reverted.